app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  validates :email, :provider, :token, presence: true
```

Comment:
Do we want to add refresh token? Or are we waiting to see which provider we're using?

Reply:
I was waiting, we can easily add validation for refresh token later.